### DISCUSSION TITLE
Website font size suddenly became really small after a new commit

### DISCUSSION THREAD

**Discussion by duckduckhero** [LekoArts/gatsby-themes#1055] [Questions]
for reference purpose, [this](https://upwind-blog.vercel.app/) is the blog i'm currently running on vercel. 

I had no problem with using the template until recently i spotted a strange error. 

After making some changes regarding the 'about' page (simply editing text content, did not touch any crucial codes or configurations) and adding a new blog post (again, followed what is written on the project README, did not make any changes to the code), I pushed this commit to my github repo.

When the build was finished on vercel, I found out that the font size suddenly shrunk and became really minimal. Did anyone run into a similar problem or know how to fix this? Would be a huge help :) 

Thank You.

**Comment by LekoArts** [comment]
https://github.com/LekoArts/gatsby-themes/discussions/1047
